### PR TITLE
[fix](move-memtable) ignore single replica load when move memtable

### PR DESCRIPTION
**Pull request by kaijchen**
## Proposed changes

Single replica load is not compatible with memtable on sink node.
Enabling single replica load when move memtable will cause missing location in OlapTableSink plan.

Ignore single replica load config when memtable on sink node is enabled.
Namely `enable_single_replica_load` config in fe.conf and `enable_single_replica_insert` in session variable.

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by kaijchen** [comment]
run buildall

**Comment by kaijchen** [comment]
run buildall

**Review by liaoxin01 — APPROVED** [review]
LGTM

File: fe/fe-core/src/main/java/org/apache/doris/analysis/NativeInsertStmt.java [hunk 1]
```diff
@@ -1023,13 +1023,16 @@ private DataSink createDataSink() throws AnalysisException {
...
+                    ? false : analyzer.getContext().getSessionVariable().isEnableSingleReplicaInsert();
             if (isGroupCommitStreamLoadSql) {
                 sink = new GroupCommitBlockSink((OlapTable) targetTable, olapTuple,
-                        targetPartitionIds, analyzer.getContext().getSessionVariable().isEnableSingleReplicaInsert(),
+                        targetPartitionIds, enableSingleReplicaLoad,
                         ConnectContext.get().getSessionVariable().getGroupCommit(), 0);
             } else {
                 sink = new OlapTableSink((OlapTable) targetTable, olapTuple, targetPartitionIds,
-                        analyzer.getContext().getSessionVariable().isEnableSingleReplicaInsert());
+                        enableSingleReplicaLoad);
```
Comment (morrySnow) [inline fe/fe-core/src/main/java/org/apache/doris/analysis/NativeInsertStmt.java:1035]: do we need add this feature to nereids?

**Review by dataroaring — APPROVED** [review]
LGTM

File: fe/fe-core/src/main/java/org/apache/doris/analysis/NativeInsertStmt.java (hunk 1 above)
Comment (kaijchen) [reply fe/fe-core/src/main/java/org/apache/doris/analysis/NativeInsertStmt.java:1035]: This is a parameter configured by the user, in this case through session variables.
What do you mean "add this feature to nereids"?

**Comment by liaoxin01** [comment]
run cloud_p1

**Comment by kaijchen** [comment]
run buildall

**Comment by liaoxin01** [comment]
run fe_ut

**Comment by liaoxin01** [comment]
run buildall

**Comment by liaoxin01** [comment]
run p0

File: fe/fe-core/src/main/java/org/apache/doris/analysis/NativeInsertStmt.java (hunk 1 above)
Comment (kaijchen) [reply fe/fe-core/src/main/java/org/apache/doris/analysis/NativeInsertStmt.java:1035]: Currently there is no `single_replica_load` option in nereids.
This PR is for fixing a problem.
If there is a need, we can create a new PR for that feature.

**Pull request merged**